thewalrus/tests/test_grouped_click_probabilities.py

```diff
@@ -0,0 +1,47 @@
+"""Test for the grouped click probability function"""
+from itertools import product
+import numpy as np
+import pytest
+from thewalrus.random import random_interferometer
+from thewalrus.symplectic import passive_transformation, squeezing
+from thewalrus.quantum import mean_clicks, variance_clicks
+from thewalrus._torontonian import threshold_detection_prob
+from grouped_click_probabilities import grouped_click_probabilities_squeezed
```

Comment:
This import statement is failing. Try something like
```suggestion
from thewalrus.grouped_click_probabilities import grouped_click_probabilities_squeezed
```